docs/knowledgebase/getting-started/glossary.md

```diff
@@ -220,80 +233,89 @@ some arguments. Such published functions take one additional parameter, known as
 
 ## Equivocating
 
-Voting or otherwise backing multiple mutually-exclusive options within the consensus mechanism. This
-is considered fundamentally Byzantine behaviour.
+A type of [Byzantine](#byzantine-fault-tolerance-bft) (erroneous/malicious) behavior that involves
+backing multiple mutually-exclusive options within the [consensus](#consensus) mechanism.
 
 ## Ethash
 
-One of a number of proofs of work, to be used in a Proof-of-Work (PoW) consensus algorithm.
-Originally developed for, and used in, Ethereum by a team led by Tim Hughes.
+A function used by some [proof-of-work](#proof-of-work) [consensus](#consensus) systems, notably
+that which is used by the Ethereum blockchain. It was developed by
+[a team led by Tim Hughes](https://github.com/ethereum/ethash/graphs/contributors).
 
 ## Events
 
-A means of recording, for the benefit of the off-chain world, that some particular state transition
-has happened. Within the context of the SRML, events are one of a number of amalgamatable datatypes
-that each module defines individually and which are aggregated together into a single overall `enum`
-type that can represent all module's types. Events are implemented through a transient set of
-storage items which are inspected immediately after a block has executed and reset during
-block-initialisation.
+A means of recording, for the benefit of the off-chain world, that some particular [state](#state)
+transition has happened. Within the context of [FRAME](#frame), events are one of a number of
+composable data types that each [pallet](#pallet) may individually define. Events in FRAME are
+implemented as a set of ephemeral storage items that are inspected immediately after a block has
+executed and reset during block-initialization.
```

Comment:
Hmmm, it's not completely wrong, but the word ephemeral evokes the wrong sense here. It's not like you can't find them anymore after the next block, you just go back to the state between blocks `n` and `n+1` to find the events from `n`.

Reply:
Okay I'm just reverting back to transient and leaving it at that...let me know if you have other specific suggestions for improvement.